Argo/Types/Decoded/Decoded.swift

```diff
@@ -82,8 +82,8 @@ public extension Decoded {
     - returns: A `Decoded.Failure` with a `.TypeMismatch` error constructed
                from the provided `expected` and `actual` values
   */
-  static func typeMismatch<T, U>(expected: String, actual: U) -> Decoded<T> {
-    return .Failure(.TypeMismatch(expected: expected, actual: String(actual)))
+  static func typeMismatch<T, U>(_ expected: String, actual: U) -> Decoded<T> {
```

Comment:
ya I agree ... I think the Swift converter in the latest Xcode switched to doing this for you so it wouldn't break changes else where ... I think this got switched because we already ran the converter with the first beta.

I'll change it back. Comment on any others you feel were better the other way.